snippets/product-card.liquid

```diff
@@ -65,7 +65,7 @@
             </div>
           </div>
         {%- else -%}
-          <div class="card__content"><h2 class="card__text h2">{{ product_card_product.title }}</h2></div>
+          <div class="card__content"><h2 class="card__text h3">{{ product_card_product.title }}</h2></div>
```

Comment:
Should we add `escape` filter as well? I'm seeing HTML tags 

![image](https://user-images.githubusercontent.com/658169/124977521-90dcaa80-dffe-11eb-98be-787511b0c9cc.png)

Reply:
This is not being rendered by the `product-card` snippet, but rather inside `main-search.liquid`
```
<h2 class="card__text h3">{{ item.title | truncate: 50 | escape }}</h2>
```
The reason the HTML tags, is because the title is being escaped. We actually would need to remove `escape` filter to fix this.

Reply:
Sounds good, thanks for sharing context! We can do this separately.

Reply:
I think we're better off outputting the html than it being added to the page as html, otherwise `</div>` in the output would just blow up the layout entirely.